pom.xml

```diff
@@ -283,88 +194,15 @@
 				</executions>
 			</plugin>
 
-			<!-- install -->
-			<plugin>
-				<artifactId>maven-install-plugin</artifactId>
-				<version>3.1.1</version>
-			</plugin>
-
-			<!-- deploy -->
-			<plugin>
-				<artifactId>maven-deploy-plugin</artifactId>
-				<version>3.1.1</version>
-			</plugin>
-
 		</plugins>
 	</build>
 
 	<reporting>
 		<plugins>
 
-			<!-- jxr: creates XRef links -->
-			<plugin>
-				<artifactId>maven-jxr-plugin</artifactId>
-				<version>3.3.2</version>
-			</plugin>
-
-			<!-- checkstyle -->
-			<plugin>
-				<artifactId>maven-checkstyle-plugin</artifactId>
-				<version>3.3.1</version>
-				<configuration>
-					<sourceEncoding>${project.build.sourceEncoding}</sourceEncoding>
-					<linkXref>true</linkXref>
-					<configLocation>checkstyle.xml</configLocation>
-				</configuration>
-			</plugin>
-
-			<!-- pmd -->
-			<plugin>
-				<artifactId>maven-pmd-plugin</artifactId>
-				<version>3.21.2</version>
-				<configuration>
-					<linkXref>true</linkXref>
-					<sourceEncoding>${project.build.sourceEncoding}</sourceEncoding>
-					<minimumTokens>50</minimumTokens>
-					<targetJdk>${maven.compiler.target}</targetJdk>
-				</configuration>
-			</plugin>
-
-			<!-- spotbugs -->
-			<plugin>
-				<groupId>com.github.spotbugs</groupId>
-				<artifactId>spotbugs-maven-plugin</artifactId>
-				<version>4.8.3.0</version>
-			</plugin>
-
-			<!-- changelog -->
-			<plugin>
-				<artifactId>maven-changelog-plugin</artifactId>
-				<version>2.3</version>
-			</plugin>
-
-			<!-- surefire -->
-			<plugin>
-				<artifactId>maven-surefire-report-plugin</artifactId>
-				<version>3.2.5</version>
-			</plugin>
-
-			<!-- javadoc -->
-			<plugin>
-				<artifactId>maven-javadoc-plugin</artifactId>
-				<reportSets>
-					<reportSet>
-						<reports>
-							<report>javadoc</report>
-						</reports>
-					</reportSet>
-				</reportSets>
-			</plugin>
-
 			<!-- Default project-info-reports -->
 			<plugin>
 				<artifactId>maven-project-info-reports-plugin</artifactId>
-				<version>3.5.0</version>
 				<reportSets>
 					<reportSet>
 						<reports>
```

Comment:
We could list all these reports in `oss-parent`. It's purpose is to have all reports expect `about`, which is overridden by `index.md` anyway, and to avoid duplicate entries in the site's left menu.